rdkit/RDConfig.py

```diff
@@ -29,10 +40,13 @@
   if condaDir[0]=='':
     condaDir[0] = os.path.sep
   condaDir += ['share','RDKit']
-  _share = os.path.join(*condaDir)
-  RDDataDir=os.path.join(_share,'Data')
-  RDDocsDir=os.path.join(_share,'Docs')
-  RDProjDir=os.path.join(_share,'Projects')
+  RDBaseDir = os.path.join(*condaDir)
+  RDDataDir=os.path.join(RDBaseDir,'Data')
+  RDDocsDir=os.path.join(RDBaseDir,'Docs')
+  RDProjDir=os.path.join(RDBaseDir,'Projects')
+  RDBinDir=MissingCondaDir('bin')
+  RDDemoDir=MissingCondaDir('Demo')
```

Comment:
I don't think either RDDemoDir or RDBinDir are still needed. Probably makes more sense to just leave them out (and remove them from the other paths through RDConfig.py).